socketio/src/client/builder.rs

```diff
@@ -113,6 +115,11 @@ impl ClientBuilder {
         self
     }
 
+    pub fn reconnect_on_disconnect(mut self, reconnect_on_disconnect: bool) -> Self {
```

Comment:
This is a public method, could you please add a doc comment describing the behaviour and potentially an example for this? :)